content/en/blog/falcosidekick-reponse-engine-part-8-fission.md

```diff
@@ -0,0 +1,505 @@
+---
+title: "Kubernetes Response Engine, Part 8: Falcosidekick + Fission"
+date: 2021-07-22
```

Comment:
Please update the date to a more recent one (after release of 2.24.0 of falcosidekick)